tests/tasks/test_control_flow.py

```diff
@@ -192,10 +224,6 @@ def false_branch():
         state = flow.run()
         assert state.result[merge_task].result == 0
 
-    with prefect.context(CONDITION=None):
```

Comment:
Why did this get removed?

Reply:
It used to illustrate an invalid option but with this change it evaluates False

Reply:
Ah I see